go/runtime/txpool/txpool.go

```diff
@@ -333,6 +338,16 @@ func (t *txPool) FinishScheduling() {
 	t.drainLock.Unlock()
 }
 
+func (t *txPool) RejectTxs(hashes []hash.Hash) {
+	// Remove rejected transactions from the already seen cache to allow them to be resubmitted as
+	// they may become valid in the future.
+	for _, h := range hashes {
+		t.seenCache.Remove(h)
+	}
+
+	t.HandleTxsUsed(hashes)
```

Comment:
A bit confusing... Are rejected transactions "processed in a block" (description of `HandleTxsUsed`)?

Reply:
No, but the logic is the same (e.g. they should be removed from queue). Maybe the naming is a bit awkward.